scripts/eurostat/health_determinants/Alcohol_Consumption/process.py

```diff
@@ -0,0 +1,749 @@
+        'DEG1':'Cities',
+        'DEG2':'TownsAndSuburbs',
+        'DEG3':'RuralAreas',
+        }})
+    return df
+
+def _replace_c_birth(df:pd.DataFrame) -> pd.DataFrame:
+    """
+    Replaces values of a single column into true values
+    from metadata returns the DF
+    """
+    df = df.replace({'c_birth': {
+        'EU28_FOR': 'ForeignBornWithinEU28',
+        'NEU28_FOR': 'ForeignBornOutsideEU28',
+        'FOR': 'ForeignBorn',
+        'NAT': 'Native'
+    }})
+    return df
+
+def _replace_citizen(df:pd.DataFrame) -> pd.DataFrame:
+    """
+    Replaces values of a single column into true values
+    from metadata returns the DF
+    """
+    df = df.replace({'citizen': {
+        'EU28_FOR': 'ForeignWithinEU28',
+        'NEU28_FOR': 'ForeignOutsideEU28',
+        'FOR': 'NotACitizen',
+        'NAT': 'Citizen'
+    }})
+    return df
+
+class EuroStatAlcoholConsumption:
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+    def __init__(self, input_files: list, csv_file_path: str,\
+        mcf_file_path: str, tmcf_file_path: str) -> None:
+        self.input_files = input_files
+        self.cleaned_csv_file_path = csv_file_path
+        self.mcf_file_path = mcf_file_path
+        self.tmcf_file_path = tmcf_file_path
+
+    def _generate_tmcf(self) -> None:
+        """
+        This method generates TMCF file w.r.t
+        dataframe headers and defined TMCF template.
+        Arguments:
+            None
+        Returns:
+            None
+        """
+        tmcf_template = (
+            "Node: E:EuroStat_Population_AlcoholConsumption->E0\n"
+            "typeOf: dcs:StatVarObservation\n"
+            "variableMeasured: C:EuroStat_Population_AlcoholConsumption->SV\n"
+            "measurementMethod: C:EuroStat_Population_AlcoholConsumption->"
+            "Measurement_Method\n"
+            "observationAbout: C:EuroStat_Population_AlcoholConsumption->geo\n"
+            "observationDate: C:EuroStat_Population_AlcoholConsumption->time\n"
+            "value: C:EuroStat_Population_AlcoholConsumption->observation\n")
+        # Writing Genereated TMCF to local path.
+        with open(self.tmcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(tmcf_template.rstrip('\n'))
+
+    def _generate_mcf(self, sv_list) -> None:
+        """
+        This method generates MCF file w.r.t
+        dataframe headers and defined MCF template
+        Arguments:
+            df_columns (list) : List of DataFrame Columns
+        Returns:
+            None
+        """
+        # pylint: disable=R0914
+        # pylint: disable=R0912
+        # pylint: disable=R0915
+        mcf_template = ("Node: dcid:{}\n"
+                        "typeOf: dcs:StatisticalVariable\n"
+                        "populationType: dcs:Person{}{}{}{}{}{}{}{}\n"
+                        "statType: dcs:measuredValue\n"
+                        "measuredProperty: dcs:count\n")
```

Comment:
Please use named arguments instead of positional arguments.

Reply:
Done the change